networkit/base.pyx

```diff
@@ -43,10 +47,13 @@ cdef class Algorithm:
 		return self._this.hasFinished()
 
 	def toString(self):
-		""" Get string representation.
+		""" 
+		toString()
+
+		Get string representation.
 
-		Returns:
-		--------
+		Returns
+		-------
 		string
 			String representation of algorithm and parameters.
```

Comment:
This generates conflicts with #864. Since `toString` and `isParallel` (see below) will be removed soon, we can avoid these changes.